Users/Install.rst

```diff
@@ -106,6 +106,13 @@ folder:
 
     $ virtualenv venv
 
+.. note::
+
+    If you have both Python3 and Python2 installed, use
```

Comment:
You should add a space between "Python" and its version, so it doesn't count as an own name.

Reply:
Dont we always write it as `Python3 and Python2`?
P.S it was separated earlier but i asked to remove the space ;P